CI/flatpak/com.obsproject.Studio.json

```diff
@@ -409,10 +409,63 @@
         }
       ]
     },
+    {
+      "name": "corrosion",
+      "buildsystem": "cmake-ninja",
+      "build-options": {
+        "build-args" : [
+          "--share=network"
+        ]
```

Comment:
This should not exist, There is many rust apps that are available as Flatpak don't need such workaround.

Reply:
Fixed